core/src/main/scala/com/salesforce/op/evaluators/Evaluators.scala

```diff
@@ -85,8 +86,8 @@ object Evaluators {
         override def evaluate(dataset: Dataset[_]): Double = {
           import dataset.sparkSession.implicits._
           val dataUse = makeDataToUse(dataset, getLabelCol)
-          new MulticlassMetrics(dataUse.select(getPredictionValueCol, getLabelCol).as[(Double, Double)].rdd)
-            .precision(1.0)
+            .select(getPredictionValueCol, getLabelCol).as[(Double, Double)].rdd
+          if (dataUse.isEmpty()) 0.0 else new MulticlassMetrics(dataUse).precision(1.0)
```

Comment:
you pass this in in other places, why hardcode here?

Reply:
Fixed.